packages/eui/src/components/tabs/tabs.tsx

```diff
@@ -67,9 +75,36 @@ export const EuiTabs = forwardRef<EuiTabRef, EuiTabsProps>(
       bottomBorder && styles.bottomBorder,
     ];
 
+    const handleKeyDown = (event: KeyboardEvent) => {
+      const currentEl = elementRef.current;
+
+      const tabs = currentEl?.querySelectorAll('[role="tab"]');
```

Comment:
I actually really like querying for the `tab` role (as opposed to using our [tabbable library](https://github.com/focus-trap/tabbable). It's possible for consumers to insert wildcard children into `<EuiTabs>` (although it's not accessibly correct to do so), so this query selector ensures that only the correct tab children receive arrow key navigation.

That being said, I'd consider modifying this selector to exclude disabled and inert tabs, so they're not included in arrow key navigation:

```suggestion
      const tabs = currentEl?.querySelectorAll('[role="tab"]:not(:disabled, [inert])');
```

The tabbable library additionally checks for `visibility: hidden` or `display: none`, but IMO we don't need to go that far here for those edge cases.

Reply:
Oh, also worth noting you can avoid the `as` casting below by passing a generic to `querySelectorAll`:

```suggestion
const tabs = currentEl?.querySelectorAll<HTMLButtonElement>('[role="tab"]:not(:disabled, [inert])');
```

(prettier will probably have Opinions about splitting this into multiple lines at that point, so deliberately leaving the indentation wrong on this suggestion)

Reply:
@cee-chen oh, good to know about the [tabbable](https://github.com/focus-trap/tabbable) library, thanks, Cee! 🙌🏻  

Skipping navigation for the disabled and inert elements makes total sense 👌🏻